adam/curriculum/phase1_curriculum.py

```diff
@@ -1559,35 +1565,110 @@ def make_behind_in_front_templates() -> Iterable[Phase1SituationTemplate]:
     )
 
 
+# Simple Pursuit curriculum
+def _make_simple_pursuit_curriculum(
+    n_exemplar: int = 10, n_noise: int = 0, n_objects_in_scene: int = 3
+) -> Phase1InstanceGroup:
+    if n_noise > n_exemplar:
+        raise RuntimeError("Cannot have more noise than regular exemplars")
+
+    target_objects = [BALL, CHAIR, PERSON, TABLE, DOG, BIRD, BOX]
+    target_object_variables = [
+        object_variable(target.handle + "-target", target) for target in target_objects
+    ]
+    other_objects = [
+        object_variable("obj-" + str(idx)) for idx in range(n_objects_in_scene - 1)
```

Comment:
This notation will pull liquids and body parts as possible other objects. Prefer `standard_object` if we want to exclude those object types.